docs/host_limit.rst

```diff
@@ -17,3 +17,32 @@ Event data example:
 The value for "hosts" can be a comma delimited string of multiple host names, but typically
 it is a single host. This is useful to restrict a remedy playbook to run only on the problematical host
 that emits a monitored event. Be sure that the source plugin already sends this metadata.
+
+
+Inserting hosts to meta
+-----------------------
+
+The plugins may not send metadata like for example the `ansible.eda.webhook` plugin which
+sends the arbitrary payload under the `payload` key.
+
+To accommodate this, the EDA collection provides the `insert_hosts_to_meta` filter,
+allowing any plugin to customize the value of `event.meta.hosts` based on the contents
+of a specific key in the event.
+
+.. note::
+
+    The `host_path` in filter `insert_hosts_to_meta` don't expect the `event` root key unlike conditions
+    or action arguments and it won't take action if the `host_path` is not found in the event.
+
+
+Example:
+
+.. code-block:: yaml
+
+    sources:
+        - ansible.eda.webhook:
+            port: 4444
+            host: 0.0.0.0
```

Comment:
@Alex-Izquierdo The newer webhook plugin has been defaulting the host to 0.0.0.0 so we can skip it here.

Reply:
Removed